src/models/Device.ts

```diff
@@ -139,59 +140,66 @@ export class NrfDevice {
 		});
 	}
 
-	async initShadow(appVersion: string = ''): Promise<void> {
-		await this.publish(this.topics.shadow.update._, {
-			state: {
-				reported: {
-					device: {
-						serviceInfo: {
-							fota_v2: ['APP', 'MODEM'],
-							ui: [
-								'GPS',
-								'FLIP',
-								'TEMP',
-								'HUMID',
-								'AIR_PRESS',
-								'BUTTON',
-								'LIGHT',
-								'LOG',
-							],
+	//Change this to handle atv2 and mss.
```

Comment:
Sounds like a TODO, which it's not. 
```suggestion
	// Create a shadow for either Asset Tracker v2 (atv2) or Multi-Service Sample (mss)
```